.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "fortio/cert-manager"]
```

Comment:
@whereisaaron while I have you - is there any way to get
```
helm install --name cert-manager cert-manager/contrib/charts/cert-manager/
```
without downloading the whole git repo for just the couple of files inside contrib/charts ?

Reply:
If you download the .tgz you can just install locally. You create the tgz with `helm package`
```
cd cert-manager/contrib/charts/
helm package cert-manager
```
The copy the tgz somewhere and
```
helm install --name cert-manager cert-manager-*.tgz
```
If you would it to be a download, you need it make a repository, which is just a folder on a website with the `helm package` files in it, along with an `index.yaml`, which you make by pointing [`helm repo index`](https://docs.helm.sh/helm/#helm-repo-index) at the folder.

Then you can `helm repo add` your folder URL, and `helm install` from it.
```
helm repo add https://example.com/my-charts
helm install --name cert-manager cert-manager
```
Where `my-charts` contains:
```
index.yaml
cert-manager-1.2.3.tgz
```

Reply:
any chance you could make that tgz part of the assets on
https://github.com/jetstack/cert-manager/releases
so I can put directly 
```
helm install --name cert-manager https://github.com/jetstack/cert-manager/releases/download/v0.2.2/helm-chart-v0.2.2.tar.gz
```
for instance?

Reply:
I intend to submit the Helm chart upstream to kubernetes/charts once https://github.com/jetstack/cert-manager/pull/229 is merged, which should clean this up for you!

Reply:
Reminds me. Need to add and extraParams option to chart for the cert manager container so we can set that default namespace option for cluster issuer secrets.

Reply:
I see https://github.com/jetstack/cert-manager/pull/229 is merged - is there a URL or short path without requiring cloning/forking/downloading now possible ?

Reply:
@munnerz I'd really love to not have that git submodule and not download anything
a working helm install https url (or base package) would be super awesome

Reply:
alternatively a simple yaml would work too

Reply:
I've created this PR upstream against k/charts: https://github.com/kubernetes/charts/pull/3416

Reply:
I saw it was merged 🎉  so now this PR is much simpler ! thanks!